bedrock/firefox/templates/firefox/features/sync.html

```diff
@@ -0,0 +1,165 @@
+  {% stylesheet 'firefox-features-hub-detail' %}
+  {% stylesheet 'firefox-features-sync' %}
+{% endblock %}
+
+{% block body_id %}firefox-features-sync{% endblock %}
+
+{% block body_class %}
+  {{ super() }}
+  state-default
+{% endblock %}
+
+{% block features_header_image %}
+  {{ high_res_img('firefox/features/sync-hero.png', {'class': 'hero-image', 'alt': '', 'width': '700', 'height': '390'}) }}
+{% endblock %}
+
+{% block features_header_content %}
+  <h1><span>{{ _('Firefox Sync') }}</span></h1>
+{% endblock %}
+
+{% block features_list %}
+<section>
+  <div class="content">
+    <h2 class="section-title"><span>{{ _('Sync') }}</span></h2>
+
+    <div id="sync-intro-copy-cta-wrapper">
+      <div id="sync-intro-copy">
+        {# Warning Text #}
+        <div class="show-fx-29-30 show-fx-28-older warning">
+          <p>
+            <span>{{ _('It looks like you’re running an older version of Firefox.') }}</span>
+          </p>
+        </div>
+
+        <div class="show-not-fx warning">
+          <p>
+            <span>
+            {% if l10n_has_tag('firefox_sync_non_fx') %}
+              {{ _('Sync is just one of the great features you’ll only get with Firefox.') }}
+            {% else %}
+              {{ _('Sync is just one of the great features you’ll only get with Firefox. Discover them all!') }}
+            {% endif %}
+            </span>
+          </p>
+        </div>
+
+      {% block intro %}
+        <div class="show-fx-31-signed-in">
+          <h1>{{ _('Ready, Set, Sync') }}</h1>
+          <p>
+            {{ _('You’re signed up and ready to access <em>your</em> Firefox anywhere, anytime.') }}
+          </p>
+        </div>
+        <div class="show-fx-31-signed-out show-fx-android show-not-fx show-default">
+          <h1>{{ _('Take your Web with you') }}</h1>
+          <p>
+            {{ _('Sync Firefox wherever you use it to access your bookmarks, passwords, tabs and more from any smartphone, tablet or computer.') }}
+          </p>
+        </div>
+      {% endblock %}
+      </div>{#--/#sync-intro-copy--#}
+
+      <div id="sync-intro-cta">
+        <div class="show-fx-31-signed-in">
+          <ul class="primary-buttons">
+            <li>
+              {{ google_play_button(id='cta-android') }}
+            </li>
+            <li>
+              <a href="{{ firefox_ios_url('mozorg-sync_page-appstore-button') }}" data-link-type="download" data-download-os="iOS" id="cta-ios">
+                <img src="{{ l10n_img('firefox/ios/btn-app-store.svg') }}" alt="{{ _('Download on the App Store') }}" width="152" height="45">
+              </a>
+            </li>
+          </ul>
+
+          <p class="button-footer">
+            <a href="https://support.mozilla.org/kb/how-do-i-set-up-firefox-sync" class="ga-link" data-interaction="outbound link click">
+              {{ _('Need help?') }}
+            </a>
+          </p>
+        </div>
+
+        <div class="show-fx-31-signed-out show-fx-android">
```

Comment:
I thought UITour was only desktop as well, but Kirby tested yesterday on Android and said it worked. I'll ask Craig to verify, just to be safe.

And yeah, I thought about the lack of iOS handling, too, but again didn't want to expose scope creep when we promised to get this out this week. (Meta problems.) I'll bring it up for the follow-up L10n related PR.